pil_analyzer/src/pil_analyzer.rs

```diff
@@ -40,60 +50,97 @@ struct PILAnalyzer<T> {
     /// appear in the source.
     source_order: Vec<StatementIdentifier>,
     symbol_counters: Option<Counters>,
-    included_files: HashSet<PathBuf>,
-    line_starts: Vec<usize>,
     current_file: PathBuf,
+    current_line_starts: Vec<usize>,
```

Comment:
what's "line starts"?

Reply:
This is a lookup data structure to help convert file positions to line-column-pairs.